packaging/nnfw.spec

```diff
@@ -14,10 +14,11 @@ Source1004: eigen.tar.gz
 Source1005: gemmlowp.tar.gz
 Source1006: ruy.tar.gz
 Source2001: nnfw.pc.in
+Source2002: nnfw-plugin.pc.in
 
 %{!?build_type:     %define build_type      Release}
 %{!?coverage_build: %define coverage_build  0}
-%{!?test_build:     %define test_build      1}
+%{!?test_build:     %define test_build      0}
```

Comment:
You'd like to disable publishing nnfw-test rpm ?

Reply:
Yes. Default disable publishing nnfw-test rpm (ex. obs build, tizen daily snapshot), but release on our repo by building with `test_build 1`.

I want your opinion.

Reply:
> Yes. Default disable publishing nnfw-test rpm (ex. obs build, tizen daily snapshot), but release on our repo by building with `test_build 1`.
> 
> 
> 
> I want your opinion.

I have no preference. Since it is test purpose, it is ok to disable publishing on  tizen public.